File: 30-max-flow/python/main.py
import json
import sys


def solve(n, edges):
    # TODO: return the maximum flow from node 1 to node n
    pass


def main():
    obj = json.load(sys.stdin)
    print(solve(obj["n"], obj["edges"]))


if __name__ == "__main__":
    main()

File: 30-max-flow/go/main.go
package main

import (
	"encoding/json"
	"fmt"
	"os"
)

type input struct {
	N     int       `json:"n"`
	Edges [][]int64 `json:"edges"`
}

func solve(n int, edges [][]int64) int64 {
	_ = n
	_ = edges
	panic("TODO")
}

func main() {
	var in input
	if err := json.NewDecoder(os.Stdin).Decode(&in); err != nil {
		fmt.Fprintln(os.Stderr, err)
		os.Exit(1)
	}
	fmt.Println(solve(in.N, in.Edges))
}
